aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -201,6 +201,17 @@ struct Vec256 {
     }
     return mask;
   }
+  Vec256<T> isnan() const {
```

Comment:
Please add test here https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/test/vec256_test_all_types.cpp

Reply:
@VitalyFedyunin  test case added in `vec256_test_all_types.cpp`, please review!